docs/src/pages/guides/notifications.md

```diff
@@ -298,6 +299,27 @@ Monika supports Discord. To enable notification via Discord, you must create a d
 | Type | Notification types                                            | `discord`                                                      |
 | Url  | The URL of the Discord Webhook that will receive notification | `https://discord.com/api/webhook/<webhook.id>/<webhook.token>` |
 
+## Facebook Workplace
+
+Monika supports Facebook Workplace. To enable notifiation via Workplace, you must create custom integration first. More info at [Facebook Workplace Custom Integrations](https://developers.facebook.com/docs/workplace/custom-integrations-new/)
+
+```json
+{
+  "id": "unique-workplace-id",
+  "type": "workplace",
+  "data": {
+    "thread_id": "12345678910",
+    "access_token": "your_custom_integration_access_token"
+  }
+}
+```
+
+| Key  | Description                                     | Example            |
+| ---- | ----------------------------------------------- | ------------------ |
+| ID   | Notification identity number                    | `Workplace12345`   |
+| Type | Notification types                              | `slack`            |
+| ID   | It's located at thread url, in the last segment | `6367478493277649` |
```

Comment:
I think the key here should be `thread_id`, right? Can you also please add an example of the thread URL?

Reply:
ah ya, it should be thread_id, I will update it. and here is the thread URL example

https://workwork.workplace.com/chat/t/6367478493277649